demos/unity_states/cs/src/BallContract.cs

```diff
@@ -76,7 +76,7 @@ ulong color
             scale_x, scale_y, scale_z,
             color
         ));
-        return new ObjectHandle(p);
+        return new ObjectHandle(p, OwnershipSemantics.SharedRef);
```

Comment:
Should be `Owned`: https://github.com/0xTELEPORTAL/constellation/blob/2f90dae624ebb56d001bc0f52674a2311c89149a/demos/unity_states/rust/src/lib.rs#L169-L182